apidoc/Annotation.yml

```diff
@@ -15,31 +15,31 @@ description: |
     The controls on the left and right side of the annotation can be specified in one of two
     ways:
 
-    * To display an image, set the [leftButton](Titanium.Map.Annotation.leftButton) or 
-      [rightButton](Titanium.Map.Annotation.rightButton) property to an image URL. (On 
-      iOS, you can also use a [SystemButton](Titanium.UI.iPhone.SystemButton) constant 
-      to use one of the native system button icons.) 
```

Comment:
Remove all the white-spaces in the docs to keep the diff to older revisions clean. You probably edited it with Atom, so you need to adjust the line-ending in the settings.